baksmali/src/main/java/org/jf/baksmali/Adaptors/Format/PackedSwitchMethodItem.java

```diff
@@ -55,11 +55,11 @@ public PackedSwitchMethodItem(MethodDefinition methodDef, int codeAddress, Packe
         targets = new ArrayList<PackedSwitchTarget>();
 
         boolean first = true;
-        int firstKey = 0;
```

Comment:
I don't like this one either. This is the constructor, and the value is being used to set the firstKey field. The ending "this.firstKey = firstKey" makes it pretty apparent what's going on.